logstreamer/filehandling.go

```diff
@@ -471,7 +475,11 @@ func (ls *LogstreamSet) ScanForLogstreams() (result []string, errors *MultipleEr
 			if err != nil {
 				errors.AddMessage(err.Error())
 				position.Reset()
+				initialTail = ls.initialTail
+			} else if position.SeekPosition == 0 {
```

Comment:
Actually, scratch that. I think it'd make more sense for the `LogstreamLocationFromFile` to accept the initialTail value as an argument. If initialTail == true and there's no journal file it should return the LogstreamLocation value corresponding to the end of the stream.